docs/core-features/workarounds.md

```diff
@@ -1,14 +1,14 @@
 # Common tricks
 
-As explained in the [Basics of FHE](fhe\_basics.md), the challenge for developers is to adapt their code to fit FHE constraints. In this document we have collected some common examples to illustrate the kind of optimization one can do to get better performance.
+This document introduces several common techniques for optimizing code to fit Fully Homomorphic Encryption (FHE) [constraints](/docs/core-features/fhe_basics.md#pbs-management). The examples provided demonstrate various workarounds and performance optimizations that you can implement while working with the **Concrete** library.
```

Comment:
The examples provided demonstrate ->  The provided examples demonstrate ?

Reply:
They're the same.

Reply:
examples provided vs provided examples, it's not the same, the adj is before the noun

Reply:
Both sentences are grammatically correct and convey the same idea, the choice between the two is more of a stylistic preference, so it doesn't have to be changed.

Reply:
really? I thought the adjective had to be before, but if you tell me it's fine, ok